packages/wonder-blocks-tabs/src/components/navigation-tab-item.tsx

```diff
@@ -89,35 +103,55 @@ export const NavigationTabItem = React.forwardRef(function NavigationTabItem(
     );
 });
 
-const underlineStyles: CSSProperties = {
-    content: '""',
-    display: "block",
-    position: "absolute",
-    bottom: 0,
-    left: 0,
-    right: 0,
-    height: sizing.size_050,
-};
-
 const styles = StyleSheet.create({
     root: {
         listStyle: "none",
         display: "inline-flex",
+        [":has(a:hover)" as any]: {
+            boxShadow: `inset 0 -${sizing.size_050} 0 0 ${semanticColor.action.secondary.progressive.hover.foreground}`,
```

Comment:
I mostly used the secondary action semantic tokens. We may need to revisit this though once Thunder Blocks theming is in place. ([related Figma thread](https://www.figma.com/design/EuFu0U7gqc1koXm8ZhlOLp?node-id=5507-1646#1181055789))

Reply:
Good catch!! Updated the hover underline to be 2px tall. I also made sure the current + hover state remains the same at 4px!